docs/cli/ilab-profile-set.md

```diff
@@ -0,0 +1,55 @@
+# `ilab profile set`
+
+### Usecase
+
+1. User needs to set:
+    a. Model Path for SDG, Eval, Serving, Training
+    b. GPUs for SDG, Eval, Serving, Training
+    c. Training Config Per-GPU (based on vram)
+2. User wants us to detect all of the above for them and provide a sane config
+
+### Workflow
+
+The user will run `ilab profile set`
+Alongside the various model paths and GPU amounts, this command will
+set the train profile for the following scenarions:
+
+    1. CPU
+    2. single consumer GPU
+    3. multi consumer GPU
+    4. single server GPU
+    5. multi server GPU
```

Comment:
Are terms like `server` and `consumer` familiar to users? I myself don't fully understand what this means.